src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IConversionExpression.vb

```diff
@@ -1061,7 +1058,9 @@ End Module]]>.Value
 IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement) (Syntax: 'Dim a As Ac ... ddressOf M2')
   IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration) (Syntax: 'a')
     Variables: Local_1: a As System.Action
-    Initializer: IOperation:  (OperationKind.None) (Syntax: 'AddressOf M2')
+    Initializer: IDelegateCreationExpression (OperationKind.DelegateCreationExpression, Type: System.Action) (Syntax: 'AddressOf M2')
```

Comment:
>Initializer: IDelegateCreationExpression (OperationKind.DelegateCreationExpression, Type: System.Action) (Syntax: 'AddressOf M2') [](start = 4, length = 129)

Should this node be implicit?

Reply:
It should be. Added a test to verify just the initializer section.

---
In reply to: [138741293](https://github.com/dotnet/roslyn/pull/22056#discussion_r138741293) [](ancestors = 138741293)